test/mocks/server/admin.cc

```diff
@@ -13,6 +14,8 @@ MockAdmin::MockAdmin() {
   ON_CALL(*this, getConfigTracker()).WillByDefault(ReturnRef(config_tracker_));
   ON_CALL(*this, concurrency()).WillByDefault(Return(1));
   ON_CALL(*this, socket()).WillByDefault(ReturnRef(socket_));
+  ON_CALL(*this, addHandler(_, _, _, _, _)).WillByDefault(Return(true));
+  ON_CALL(*this, removeHandler(_)).WillByDefault(Return(true));
```

Comment:
Are these changes necessary?

Reply:
They are if you want to add a regression test, otherwise the `example_config_test` is failing because this returns false and the assert in the TAP code hits

Reply:
Okay no problem, the connection just wasn't immediately clear

Reply:
I know, the code for the `config_validation` is a bit tricky to read and it is not obvious that it depends on these mocks